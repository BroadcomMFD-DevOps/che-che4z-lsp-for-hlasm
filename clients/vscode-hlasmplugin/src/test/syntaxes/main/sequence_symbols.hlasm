&LABEL   INSTR ('AA'.((2 EQ 2)) SLA 'AAA').SEQ

.SEQ     END
