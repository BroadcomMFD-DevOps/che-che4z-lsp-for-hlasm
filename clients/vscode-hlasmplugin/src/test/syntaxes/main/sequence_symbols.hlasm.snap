>&LABEL   INSTR ('AA'.((2 EQ 2)) SLA 'AAA').SEQ
#^^^^^^ source.hlasm support.class.hlasm variable.other.readwrite.hlasm
#      ^^^ source.hlasm
#         ^^^^^ source.hlasm entity.name.function.hlasm
#              ^ source.hlasm
#               ^ source.hlasm
#                ^ source.hlasm string.hlasm
#                 ^^^ source.hlasm string.hlasm
#                    ^ source.hlasm punctuation.definition.tag.hlasm keyword.operator.hlasm
#                     ^ source.hlasm
#                      ^ source.hlasm
#                       ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                        ^ source.hlasm
#                         ^^ source.hlasm punctuation.definition.tag.hlasm
#                           ^ source.hlasm
#                            ^ source.hlasm punctuation.definition.tag.hlasm constant.numeric.hlasm
#                             ^ source.hlasm
#                              ^ source.hlasm
#                               ^ source.hlasm
#                                ^^^ source.hlasm punctuation.definition.tag.hlasm
#                                   ^ source.hlasm
#                                    ^ source.hlasm string.hlasm
#                                     ^^^^ source.hlasm string.hlasm
#                                         ^ source.hlasm
#                                          ^^^^ source.hlasm keyword.control.hlasm
>
>.SEQ     END
#^^^^ source.hlasm keyword.control.hlasm
#    ^^^^^ source.hlasm
#         ^^^ source.hlasm entity.name.function.hlasm
>